package.json

```diff
@@ -19,7 +19,7 @@
     "test:env": "docker-compose -f docker-compose.test.yml -f docker-compose.dev.yml up --build; npm run compose:down",
     "test:compose": "docker-compose -f docker-compose.yml -f docker-compose.test.yml up --build --remove-orphans --exit-code-from=sut ; npm run compose:down",
     "test:base": "npm run lint && npm run test:build && npm run test:unit && npm run test:legacy",
-    "test": "npm run lint && npm run test:build && npm run test:node",
+    "test": "npm run lint && npm run test:build && (npm run test:node || true)",
```

Comment:
I would not do this. Let's just get rid of the current `test` and then rename `test:base` to `test`. With that can get rid of `test:node` too, and simplify the scripts. 

You'll have to change `Dockerfile.template` to use `npm run test` and update the [README](https://github.com/balena-os/balena-supervisor#testing)

Reply:
Actually let's not get rid of `test:node`, it's useful for running just the tests while developing in `test:env`

Reply:
Don't forget about this change before merging @cywang117  :)

Reply:
Done in https://github.com/balena-os/balena-supervisor/commit/6eec238ef4f3891d7dd85e58433ee78ee087b6e1!